plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/config.py

```diff
@@ -176,39 +175,39 @@ class OptunaSweeperConf:
 ConfigStore.instance().store(
     group="hydra/sweeper/sampler",
     name="tpe",
-    package="hydra.sweeper.optuna_config.sampler",
+    package="hydra.sweeper.sampler",
     node=TPESamplerConfig,
     provider="optuna_sweeper",
 )
 
 ConfigStore.instance().store(
     group="hydra/sweeper/sampler",
     name="random",
-    package="hydra.sweeper.optuna_config.sampler",
+    package="hydra.sweeper.sampler",
```

Comment:
The default package is `_group_`, which is exactly what you have here. You can now omit package the package from the store calls for the new configs.

Reply:
will do